src/components/Code.js

```
@@ -1,50 +1,50 @@
+
+  const handleCopy = () => {
+    setCopied(true);
+    setTimeout(() => setCopied(false), 2000);
+  };
+
+  const CodeStyling = ({ className, style, tokens, getLineProps, getTokenProps }) => (
+    <Card className="position-relative pe-8 mb-2">
+      <Card.Body>
+        <pre className={className} style={style}>
+          {tokens.map((line, i) => (
+            <div {...getLineProps({ line, key: i })}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)